test/performance/README.md

```diff
@@ -0,0 +1,25 @@
+# Profiles Performance Test
+
+The `profiles.jmx` [JMeter](http://jmeter.apache.org/) test will call the Profiles application for a number of different surgeries.
+Several parameters are exposed to customise the base test.
+
+
+| Parameter                         | Description                                                        | Default               |
+|:---------------------------------|:-------------------------------------------------------------------|-----------------------|
+| `hostname`                       | URL of server to test                                              | profiles.dev.beta.nhschoices.net|
```

Comment:
Good spot, it runs against staging on the triggered performance tests, I'll update the default.